src/components/QueryOutput/QueryOutput.js

```diff
@@ -851,7 +877,87 @@ export class QueryOutput extends React.Component {
 
     return undefined
   }
-
+  getTrainingData = () => {
+    const isYScaleStringColumn = !this.chartRef?.innerChartRef?.yScale.fields[0].hasOwnProperty('aggType')
```

Comment:
We actually have a state variable in here (QueryOutput) called `aggConfig` where all these agg types are stored when they change. It would be simpler to just grab it from there I think